### PR TITLE
Allow ignoring some branches via action inputs

### PR DESCRIPTION
**Pull request by rheaditi**
You can now ignore certain branches via the `skip-branches` input.

### CODE REVIEW

File: __tests__/utils.test.ts [hunk 1]
```diff
@@ -11,17 +11,29 @@ import {
 } from '../src/utils';
 import { HIDDEN_MARKER } from '../src/constants';
 
-describe('isBotPr()', () => {
-  it('should return true for dependabot PR', () => {
-    expect(isBotPr('dependabot')).toBeTruthy();
+jest.spyOn(console, 'log').mockImplementation();
```
Comment (rajanand02) [inline __tests__/utils.test.ts]: why are we doing this here?

File: __tests__/utils.test.ts [hunk 2]
```diff
@@ -11,17 +11,29 @@ import {
...
+
+describe('shouldSkipBranchLint()', () => {
+  it('should recognize bot PRs', () => {
+    expect(shouldSkipBranchLint('dependabot')).toBeTruthy();
   });
 
-  it('should return false for non bot PR', () => {
-    expect(isBotPr('feature/awesomeNewFeature')).toBeFalsy();
+  it('should handle custom ignore patterns', () => {
+    expect(shouldSkipBranchLint('bar', '^bar')).toBeTruthy();
```
Comment (rajanand02) [inline __tests__/utils.test.ts]: :heart_eyes:

File: __tests__/utils.test.ts [hunk 3]
```diff
@@ -11,17 +11,29 @@ import {
...
+    expect(shouldSkipBranchLint('foobar', '^bar')).toBeFalsy();
+
+    expect(shouldSkipBranchLint('bar', '[0-9]{2}')).toBeFalsy();
+    expect(shouldSkipBranchLint('bar', '')).toBeFalsy();
+    expect(shouldSkipBranchLint('foo', '[0-9]{2}')).toBeFalsy();
+    expect(shouldSkipBranchLint('f00', '[0-9]{2}')).toBeTruthy();
```
Comment (rajanand02) [inline __tests__/utils.test.ts]: can you add one more test for `release` and `production` branches in the pattern

File: __tests__/utils.test.ts (hunk 1 above)
Comment (rajanand02) [reply __tests__/utils.test.ts]: oh got it :+1:

Comment (rajanand02) [reply __tests__/utils.test.ts]: do we need to do this if we aren't checking for console.log execution

**Comment by rajanand02** [comment]
@rheaditi what about that weird space issue? are we not solving it?

File: __tests__/utils.test.ts (hunk 1 above)
Comment (rheaditi) [reply __tests__/utils.test.ts]: Just didn't like seeing console.logs in the output of tests :P

File: __tests__/utils.test.ts (hunk 3 above)
Comment (rheaditi) [reply __tests__/utils.test.ts]: Added!

Comment (rheaditi) [reply __tests__/utils.test.ts]: Can you check `0295c34`

Comment (rajanand02) [reply __tests__/utils.test.ts]: :medal_sports:

**Comment by rajanand02** [comment]
> what about that weird space issue? are we not solving it?

@rheaditi this issues is still there

File: src/utils.ts [hunk 4]
```diff
@@ -199,27 +216,33 @@ export const shouldUpdatePRDescription = (
...
+/**
+ * Return a safe value to output for story type.
+ * @param {StoryResponse} story
+ */
+const getEstimateForStoryType = (story: StoryResponse) => {
+  const { story_type: type, estimate } = story;
+  if (type === 'feature') {
+    return typeof estimate !== 'undefined' ? estimate : 'unestimated';
+  }
+  return 'n/a';
```
Comment (rajanand02) [inline src/utils.ts]:
```suggestion
  return 'N/A';
```

File: lib/index.js [hunk 5]
```diff
@@ -4090,26 +4106,31 @@ exports.getStoryIcon = (storyType) => {
...
+/**
+ * Return a safe value to output for story type.
+ * @param {StoryResponse} story
+ */
+const getEstimateForStoryType = (story) => {
+    const { story_type: type, estimate } = story;
+    if (type === 'feature') {
+        return typeof estimate !== 'undefined' ? estimate : 'unestimated';
+    }
+    return 'n/a';
```
Comment (rajanand02) [inline lib/index.js]: @rheaditi src is update but not the build files

**Pull request merged**